.goreleaser.yaml

```diff
@@ -28,22 +28,73 @@ builds:
       -X main.buildDate={{.Date}}
       -X main.gitDescription={{.Summary}}
 
+  - id: darwin-build
+    dir: ./cmd/chronicle
+    binary: chronicle
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    mod_timestamp: *build-timestamp
+    ldflags: *build-ldflags
+    hooks:
+      post:
+        - cmd: .tool/quill sign-and-notarize "{{ .Path }}" --dry-run={{ .IsSnapshot }} --ad-hoc={{ .IsSnapshot }} -vv
+          env:
+            - QUILL_LOG_FILE=/tmp/quill-{{ .Target }}.log
+
+  - id: windows-build
+    dir: ./cmd/chronicle
+    binary: chronicle
+    goos: [windows]
+    goarch: [amd64]
+    mod_timestamp: *build-timestamp
+    ldflags: *build-ldflags
+
+
 nfpms:
-  -
-    formats:
-      - deb
-      - rpm
-    vendor: "anchore"
+  - vendor: "anchore"
     homepage: "https://github.com/anchore/chronicle"
     maintainer: "Alex Goodman <alex.goodman@anchore.com>"
     description: "A fast changelog generator sourced from PRs and Issues"
     license: "Apache-2.0"
-    file_name_template: "{{.ProjectName}}_{{.Version}}_{{.Os}}-{{.Arch}}"
-    replacements:
-      amd64: 64bit
-      arm64: ARM64
-      darwin: macOS
-      linux: Linux
-
-universal_binaries:
-  - replace: true
+    formats:
+      - deb
+      - rpm
+
+
+archives:
+  - id: linux-archives
+    builds:
+      - linux-build
+
+  # note: the signing process is depending on tar.gz archives. If this format changes then .github/scripts/apple-signing/*.sh will need to be adjusted
+  - id: darwin-archives
+    builds:
+      - darwin-build
+
+sboms:
+  - artifacts: archive
+    # this is relative to the snapshot/dist directory, not the root of the repo
+    cmd: ../.tool/syft
+    documents:
+      - "{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}.sbom"
+    args:
+      - "scan"
+      - "$artifact"
+      - "--output"
+      - "json=$document"
+
+signs:
```

Comment:
if we start signing we'll need to add the id token permission to the release workflow https://github.com/anchore/syft/blob/6b48bd4b5e16fa1c1a783b889b5ada52153dc06d/.github/workflows/release.yaml#L113-L117